objects/src/constants.rs

```diff
@@ -27,6 +27,14 @@ pub const BATCH_NOTE_TREE_DEPTH: u8 = 10;
 
 /// The maximum number of notes that can be created in a single batch.
 pub const MAX_NOTES_PER_BATCH: usize = 1 << BATCH_NOTE_TREE_DEPTH;
+const _: () = assert!(MAX_NOTES_PER_BATCH >= MAX_OUTPUT_NOTES_PER_TX);
+
+/// The maximum number of input notes that can be consumed in a single batch.
+pub const MAX_INPUT_NOTES_PER_BATCH: usize = MAX_NOTES_PER_BATCH;
+const _: () = assert!(MAX_INPUT_NOTES_PER_BATCH >= MAX_INPUT_NOTES_PER_TX);
```

Comment:
Let's rename `MAX_NOTES_PER_BATCH` into `MAX_OUTPUT_NOTES_PER_BATCH` to keep it parallel with `MAX_INPUT_NOTES_PER_BATCH`